bentoml/server/api_server.py

```diff
@@ -468,7 +469,7 @@ def handle_ml_processing(self):
                     'type': 'River'
                 }
             ],
-            'watchedAttributes': ['precipitation'],
+            'watchedAttributes': [ATTRIBUTE_INPUT_DATA],
```

Comment:
be careful, `precipitation` is still hardcoded in some other places in this file.

Reply:
A different route. I don't know how to make it not hardcoded. Conf ? Querying MLModel if you stand by your first comment on the watched attributes being defined by the MLModel ?

Reply:
Put it in conf, in the same way than the id of your ML model entity is configured. This is part of the definition of the ML model that is embedded in Bento, so this is not subject to change dynamically.